config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get "password_resets/new"
```

Comment:
is this line also necessary?
if yes, why? if no, why?

Reply:
I fixed.It's not necessary . Because Rails create RESTful routes via resource method like this case : 
resources :password_resets, only: %i(new create edit update)